### PR TITLE
Add the ability to change the bot's prefix

### PR DESCRIPTION
**Pull request by alazymeme**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

### CODE REVIEW

**Comment by alazymeme** [comment]
Yes I fucked up, accidentally used the `fix-pleblist` branch instead of the correct one

**Comment by alazymeme** [comment]
If it tells me to reformat again, I'm gonna kill someone

File: pajbot/dispatch.py [hunk 1]
```diff
@@ -32,7 +31,7 @@ def add_command(bot, source, message, event, args):
         # Make sure we got both an alias and a response
         message_parts = message.split()
         if len(message_parts) < 2:
-            bot.whisper(source, "Usage: !add command ALIAS [options] RESPONSE")
+            bot.whisper(source, f"Usage: {self.prefix}add command ALIAS [options] RESPONSE")
```
Comment (RAnders00) [inline pajbot/dispatch.py]: Should be bot.prefix (and in several other places too)

File: pajbot/dispatch.py [hunk 2]
```diff
@@ -19,8 +19,7 @@ def add_win(bot, source, message, event, args):
         bot.kvi["br_wins"].inc()
         bot.me(f"{source} added a BR win!")
 
-    @staticmethod
-    def add_command(bot, source, message, event, args):
+    def add_command(self, bot, source, message, event, args):
```
Comment (RAnders00) [inline pajbot/dispatch.py]: Add @staticmethod back since self is not needed

File: pajbot/managers/command.py [hunk 3]
```diff
@@ -115,7 +115,7 @@ def load_internal_commands(self):
                 CommandExample(
                     None,
                     "Default usage",
-                    chat="user:!twitterfollow forsensc2\n" "bot>user:Now following ForsenSC2",
+                    chat=f"user:{self.prefix}twitterfollow forsensc2\n" "bot>user:Now following ForsenSC2",
```
Comment (RAnders00) [inline pajbot/managers/command.py]: Same here, you need bot.prefix

**Comment by alazymeme** [comment]
@RAnders00 what should I do here: https://github.com/alazymeme/pajbot/blob/master/pajbot/web/routes/api/commands.py#L161 in order to "import" the prefix (using the bot definition or whatever it's called fdm)

**Comment by RAnders00** [comment]
```python
from flask import current_app as app

log_msg = f'The {app.bot_prefix}{command.command.split("|")[0]} command has been updated from "{old_message}" to "{new_message}"'
```

And in `web/__init__.py`, add a config parsing entry like you added in `bot.py`, this time call it `app.bot_prefix = ...` similar to how the other existing ones are already done:

https://github.com/pajbot/pajbot/blob/master/pajbot/web/__init__.py#L115

**Comment by alazymeme** [comment]
@RAnders00 Check the above commit, is this required in every file that I cannot "import" bot?

**Comment by RAnders00** [comment]
In module files the bot is available via `self.bot`.

The only pöaces you need to do it through `app` is in the web process which is in the .py files in the `web` directory.

**Comment by alazymeme** [comment]
@pajlada You haven't commented on this, so I just have to tag you and ask for your review

**Pull request closed without merging**